src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -398,6 +391,13 @@ CACHES = {
     'redis': redis_cache,
 }
 
+CELERY_BROKER_URL = BROKER_URL = '{{ BROKER_URL }}'
+CELERY_RESULT_BACKEND = redis_cache['LOCATION']
```

Comment:
@dannyroberts Figured now would be the right time to switch over to using redis as the results backend, so we don't have to change the backend more than once.  I've tested that this works locally and on staging.

Reply:
If this looks like a reasonable change to you, then I think I will break this PR into two pieces with two changelogs - one for localsettings changes (no CCHQ min version, max version prior to celery 4.x upgrade, brief service interruption when backend changes - to be performed on weekend), and one for bash script changes (min version: https://github.com/dimagi/commcare-hq/pull/23319, max version prior to celery 4.x upgrade, no service interruption - can run anytime).

Reply:
separate PRs are so the localsettings changes don't get run accidentally during peak hours, and so I can run the bash script updates sooner to avoid making too many changes at once